corehq/apps/domain/management/commands/delete_domain.py

```diff
@@ -10,7 +10,7 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument(
-            'domain',
+            '--domain',
```

Comment:
it might be nicer to just remove the `dest` and rename the argument to `domain_name`, since this isnt actually an optional argument